content/_code-samples/escrow/js/finish-escrow.js

```diff
@@ -0,0 +1,94 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // Construct an EscrowFinish transaction to finish an Escrow which has passed its FinishAfter due date
+  // https://xrpl.org/escrow.html#escrow
+  // https://xrpl.org/escrowfinish.html#escrowfinish
+
+  async function main() {
+    // To finish an Escrow, you'd need to have an account which has sent an Escrow with a 'FinishAfter' field pass its time
+    // You could use 'create_escrow.js' on the same folder to import an account
+
+    seed = ""
+
```

Comment:
`finish-escrow.js` and `cancel-escrow.js` should both have nice errors if no seed is provided to guide the user to input the correct information.